src/renderer/actions/databases.js

```diff
@@ -60,6 +64,35 @@ export function saveDatabaseDiagram(diagramJSON) {
   };
 }
 
+export function exportDatabaseDiagram(diagram, imageType) {
+  return async (dispatch) => {
+    dispatch({ type: EXPORT_DIAGRAM_REQUEST });
+    try {
+      // fix - reapply css roles which html2canvas ignores for some reason
+      $('.link-tools, .marker-arrowheads', diagram).css({ display: 'none' });
+      $('.link, .connection', diagram).css({ fill: 'none' });
```

Comment:
Feels quite weird accessing the DOM inside the redux actions. Isn't this something you could do from where you are calling this function and just pass the result as argument to exportDatabaseDiagram? Maybe I'm wrong, but I have never seen this approach before.